scheduled-jobs/build/oc-mirror-quay-dev/Jenkinsfile

```diff
@@ -23,7 +23,10 @@ properties(
                     name: 'BUILD_VERSION',
                     description: 'OCP Version to build',
                     $class: 'hudson.model.ChoiceParameterDefinition',
-                    choices: "4.0",
+                    choices: [
```

Comment:
Pull this from commonlib like we do for the rest of the jobs.

Reply:
Totes. Good point. Although I probably should have just ignored updating this file since I'm technically deprecating it in favor of the one I copied it to and updated.